### PR TITLE
Management API to push documents for user

### PR DESCRIPTION
**Pull request by ec**

### CODE REVIEW

File: spec/api/v2/management/documents_spec.rb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+    it 'push documents to user' do
+      post_json '/api/v2/management/documents', multisig_jwt_management_api_v2({ data: params }, *signers)
+
+      expect(response.status).to eq 201
+      expect(user.documents.length).to eq(1)
+    end
+
+    it 'push documents to user twice' do
+      params = {
```
Comment (ec) [inline spec/api/v2/management/documents_spec.rb:72]: move params to :let

File: spec/api/v2/management/documents_spec.rb [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+describe API::V2::Management::Documents, type: :request do
+  before do
+    defaults_for_management_api_v2_security_configuration!
+    management_api_v2_security_configuration.merge! \
+      scopes: {
+        write_documents: { permitted_signers: %i[alex jeff], mandatory_signers: %i[alex] },
+      }
+  end
+  let!(:create_admin_permission) do
```
Comment (ec) [inline spec/api/v2/management/documents_spec.rb:13]: there is no need to have admin permission

File: app/api/v2/management/documents.rb [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        post do
+          user = User.find_by!(uid: params[:uid])
+          error!(errors: ['user doesnt exist']) unless user
+
+          file = parse_file_data(params[:upload], params[:filename], params[:file_ext])
+
+          doc = user.documents.new(declared(params).except(:upload, :uid, :filename, :file_ext).merge(upload: file))
+          error!(doc.errors.full_messages.to_sentence, 422) unless doc.save
+
+          status 201
```
Comment (ec) [inline app/api/v2/management/documents.rb:62]: need to handle Carrier wave error

File: app/api/v2/management/documents.rb [hunk 4]
```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+module API::V2
+  module Management
+    # Documents server-to-server API
+    class Documents < Grape::API
+      desc 'Documents related routes'
+      resource :documents do
+        helpers do
+          def parse_file_data(upload, name, ext)
```
Comment (ec) [inline app/api/v2/management/documents.rb:10]: Can keep it like this, but better find more elegant solution

**Review by mod — APPROVED** [review]

**Pull request merged**